HHS/OPRE-OPS
# Task 819

## What changed

 - Refactor `PreviewTable` to add any budget lines from `AgreementSelect`
 - Update edit icons to only show duplicate

## Issue

#819 

## How to test

1. Select `Using Innovative Data to Explore Racial and Ethnic Disparities in Human Services` Agreement which has some budget lines.
1. See them in the table.
1. Hover over them and will only be able to duplicate them.


## Screenshots

<img width="1003" alt="image" src="https://user-images.githubusercontent.com/4629398/230666128-f96bd73c-5a1c-42e8-809c-a89226d18b45.png">
